tests/networkpolicy_test.go

```diff
@@ -0,0 +1,90 @@
+ipackage tests_test
+
+import (
+	"flag"
+	"fmt"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	expect "github.com/google/goexpect"
+
+	k8sv1 "k8s.io/api/core/v1"
+	v13 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	ktests "kubevirt.io/kubevirt/tests"
+	tests "kubevirt.io/kubevirt-ansible/tests/framework"
+	v1 "kubevirt.io/kubevirt/pkg/api/v1"
+
+	"kubevirt.io/kubevirt/pkg/kubecli"
+	"k8s.io/apimachinery/pkg/api/resource"
+)
+
+const (
+	privilegedPodYaml  = "tests/manifests/privileged-pod.yml"
+        denyallYaml  = "tests/manifests/networkpolicy-deny-all.yml"
+	allowhttpYaml = "tests/manifests/networkpolicy-allow-http.yml"
+)
+
+
+var _ = Describe("[rfe_id:150][crit:high][vendor:cnv-qe@redhat.com][level:component]Networkpolicy", func() {
+	flag.Parse()
+
+	virtClient, err := kubecli.GetKubevirtClient()
+	ktests.PanicOnError(err)
+
+	var vmia *v1.VirtualMachineInstance
+	var vmib *v1.VirtualMachineInstance
+
+	ktests.BeforeAll(func() {
+		ktests.SkipIfUseFlannel(virtClient)
```

Comment:
I would add a documentation comment, to explain why Flannel networking model should be skipped, because as opposed to *SkipIfNotUseNetwotkPolicy* - this is not self-explanatory.
Something brief and simple, like:
`// Flannel networking model doesn't support network policy`
(I had to conclude that myself from [https://github.com/coreos/flannel#networking-details](url))

Reply:
Added a comment in the step.

Reply:
Why we need both `ktests.SkipIfUseFlannel` and `ktests.SkipIfNotUseNetworkPolicy` ?

Reply:
Because networkpolicy is not supported when using flannel or openshift other networkplugin, eg, redhat/openshift-ovs-subnet or redhat/openshift-ovs-multutenant. It only works on redhat/openshift-ovs-networkpolicy.

We could check the current on cluster: 
```
# oc get clusternetwork
NAME      CLUSTER NETWORK   SERVICE NETWORK   PLUGIN NAME
default   10.128.0.0/14     172.30.0.0/16     redhat/openshift-ovs-networkpolicy
```

Reply:
ok so you are going to test it on k8s with other network cni plugins?

Reply:
Actually OpenShift can use flannel as well, but seems we don't setup OCP + flannel env for our testing. Do you want me to remove it?